src/coreComponents/fileIO/Outputs/VTKOutput.cpp

```diff
@@ -101,6 +106,7 @@ void VTKOutput::postInputInitialization()
   m_writer.setFieldNames( m_fieldNames.toViewConst() );
   m_writer.setLevelNames( m_levelNames.toViewConst() );
   m_writer.setOnlyPlotSpecifiedFieldNamesFlag( m_onlyPlotSpecifiedFieldNames );
+  m_writer.setNumberOfTargetProcesses( m_numberOfTargetProcesses );
```

Comment:
Valid values seem to be between 1 & MPI::commSize(), what do you think of adding `GEOS_ERROR()`s or at least `GEOS_WARNING()`s.
Source: https://vtk.org/doc/nightly/html/classvtkAggregateDataSetFilter.html#:~:text=void%20vtkAggregateDataSetFilter%3A%3ASetNumberOfTargetProcesses